### DISCUSSION TITLE
Double tap to go next 10 seconds or so on mobile devices

### DISCUSSION THREAD

**Discussion by uditmore99** [openplayerjs/openplayerjs#389] [Q&A] [answered]
I am implementing the player to watch m3u8 videos. I was thinking to integrate double tap to go next 10 seconds or so only on mobile devices. How can I do so ?
I am using React/NextJS as my primary techstack
Any kind of help is appreciated.
Thanks!

**Comment by rafa8626** [comment, marked as answer]
@uditmore99 sorry for my late response; I created 2 CodePens that could help you: one is using an additional element, and another with the `video` tag we already have. Test them and see which one fits your needs better (you may need to tweak this to only use mobile devices, since the `touchstart` event is for any device with that capability, including certain computers):

- https://codepen.io/rafa8626/pen/XWoKXev
- https://codepen.io/rafa8626/pen/JjwKYgg

**Reply by uditmore99** [reply]
Thank you very much! 
This solved my problem
I am using the second one with the video tag as it was more convenient

**Reply by rafa8626** [reply]
Glad it worked for you! :)
